search-commons/src/test/resources/datasource_urls_ticket.json

```diff
@@ -1,36 +1,36 @@
 {
-  "page=0&size=21": 21,
+  "page=0&size=21": 19,
```

Comment:
Each time you change something; this needs to be changed as well?

I can see the pain and suffering from this file for you guys that work with this.

Reply:
I needed to test a filter , and added it, filtering away two documents, som tests didn't include it some did.
What I gain is a real life "database" with consequences for changing data and/or logic, as in the real world.